### DISCUSSION TITLE
Get the remaining ticks before the entity stops burning

### DISCUSSION THREAD

**Discussion by Ynverxe** [Minestom/Minestom#1956] [Help]
Hi! I need to get the remaining fire ticks of an entity, this is the equivalent to the Entity#getFireTicks() of the Bukkit API. But I see that the field that holds this value is not exposed.

**Comment by GreatWyrm** [comment]
This is currently not supported in Minestom, and would require a PR to fix.

The method would look along the lines of:

```
    public int getFireTicks() {
        long difference = fireExtinguishTime - System.currentTimeMillis();
        if (difference < 0) {
            return 0;
        } else {
            return (int) Math.round((double)difference / MinecraftServer.TICK_MS);
        }
    }
```
